crates/holochain_state/src/query.rs

```diff
@@ -162,9 +162,9 @@ pub trait Store {
     /// The second parameter determines whether the warrant op should be checked for validity.
     /// It should be set to false if reading from an Authored DB, where everything is valid,
     /// and true if reading from a DHT DB, where validation status matters
-    fn get_warrants_for_basis(
+    fn get_warrants_for_agent(
```

Comment:
Do we remember the reason why this wasn't always just an agent key? Was it just future thinking that we might warrant something else?

I'd expect the warrant to always target the agent that did something to get warranted... So I can't see what we'd have been planning here.

Reply:
Was it related to the "sticky warrants" concept where we wanted to retrieve the warrant along with the action it applied to? https://github.com/holochain/holochain/issues/3641

Reply:
Maybe, yeah.

Reply:
I say if there isn't a single test to break when changing that then we treat it as though it never got implemented... I think that's right. When we retrieve an action, we want the warrant to come back with it in some cases but that should have been tested.